packages/cli/src/handlers/preview.ts

```diff
@@ -256,7 +265,11 @@ export const startPreviewHandler = async (
                 name: options.name,
             },
         });
-        await deploy({ ...options, projectUuid: project.projectUuid });
+        await deploy({
+            ...options,
+            projectUuid: project.projectUuid,
+            ignoreErrors: false,
```

Comment:
just trying to understand, should this also take into account `--ignore-errors` arg like you do in deploy?

Reply:
I see, cool 👍🏻